### PR TITLE
Add method to Session to get only enabled applications

### PR DESCRIPTION
**Pull request by gcrone**

### CODE REVIEW

**Review by glehmannmiotto — APPROVED** [review]
This looks OK. I noticed that daqconf is not using the python bindings of confmodel algorithms, but rather re-implements the algorithm (get_all_applications) in python. Is there a reason for this?
Once this PR is merged we should review all the appmodel algorithms to use the get_enabled_applications method instead of get_all_applications (also dfmodules is affected).

**Comment by gcrone** [comment]
I suspect that Eric didn't realise that there was a get_all_applications python binding.  We could also now update the oks parser in `drunc` to use get_enabled_applications.

**Pull request merged**
